tfx_addons/sampling/example/sampler_pipeline_local.py

```diff
@@ -0,0 +1,167 @@
+# Copyright 2021 The TensorFlow Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+# ==============================================================================
+"""Sampling pipeline example using TFX."""
+
+from __future__ import absolute_import, division, print_function
+
+import os
+from typing import List, Text
+
+import absl
+import tensorflow_model_analysis as tfma
+from tfx.components import (CsvExampleGen, Evaluator, ExampleValidator, Pusher,
```

Comment:
From what I can tell, tfx style is to list each import statement on its own line, for example, https://github.com/tensorflow/tfx/blob/master/tfx/examples/ranking/ranking_pipeline.py.